aggregator_core/src/datastore.rs

```diff
@@ -4583,34 +4584,38 @@ WHERE task_id = $1
     //    aggregations in the batch which are in a non-failure state (START/WAITING/FINISHED).
     async fn read_batch_size(
         &self,
-        task_id: &TaskId,
+        task_pkey: i64,
         batch_id: &BatchId,
     ) -> Result<RangeInclusive<usize>, Error> {
-        // TODO(#1467): fix this to work in presence of GC.
         let stmt = self
             .prepare_cached(
                 "-- read_batch_size()
-WITH batch_report_aggregation_statuses AS
-    (SELECT report_aggregations.state, COUNT(*) AS count FROM report_aggregations
-     JOIN aggregation_jobs
+WITH report_aggregations_count AS (
+    SELECT COUNT(*) as count FROM report_aggregations
```

Comment:
```suggestion
    SELECT COUNT(*) AS count FROM report_aggregations
```

super-nit: our current standard is that SQL keywords are capitalized, let's stick to that for now.

Reply:
Whoops!